### DISCUSSION TITLE
500 Internal Error on API key view

### DISCUSSION THREAD

**Discussion by gabrielbasetti** [appwrite/appwrite#5130] [Q&A]
Good evening, for days I have been trying to launch appwrite on my server to transfer an app from firebase, but I have this problem.
Even though the installation via docker is fine, I create a new project, create a new API key, but when I try to view the key it gives me the above mentioned error: "500 Internal Error".
I attach some images:
![image](https://user-images.githubusercontent.com/39503933/219251105-add50200-fb44-41d0-86f0-3d79350a208a.png)
![image](https://user-images.githubusercontent.com/39503933/219251142-1ec4b256-8414-49ad-8bf5-2eb344947c62.png)

Esto es _docker compose logs appwrite_ al momento de hacer click allí: 
![image](https://user-images.githubusercontent.com/39503933/219251330-87436bd0-28e0-4fa6-8a85-1a62f634583a.png)

**Comment by joeyouss** [comment]
Hi 👋🏻 can your please share the complete error logs so I can see what might be causing the problem?

**Reply by stnguyen90** [reply]
@gabrielbasetti, specifically, please share the logs for the `appwrite` container. You're only sharing a small snippet of logs from the `appwrite-influxdb` and `appwrite-usage` containers.

**Comment by gabrielbasetti** [comment]
Here it is.
Also i've forgot to paste doctor's output:
![image](https://user-images.githubusercontent.com/39503933/219663366-923f56bf-172e-4dbf-be96-977cdbc21d71.png)


[appwrite_docker_logs.txt](https://github.com/appwrite/appwrite/files/10767125/appwrite_docker_logs.txt)

Thank you very much!

**Reply by stnguyen90** [reply]
@gabrielbasetti, unfortunately, the logs you shared don't have the 500 error. Would you please reproduce the error and then grab the logs again? All we need are the logs from the `appwrite` container, which can be retrieved by running `docker compose logs appwrite`.

Also, it might be better to troubleshoot this on [Discord](https://appwrite.io/discord) rather than in a GitHub discussion.

**Comment by gabrielbasetti** [comment]
Well,
I kept investigating and came to the conclusion that the problem comes from the reverse proxy.
I use Cyberpanel OpenLiteSpeed as hosting server, on it I mounted the appwrite container and used this guide to get external access to the container: https://community.cyberpanel.net/t/reverse-proxy-traffic-to-docker-container-on-cyberpanel/30644
Everything works fine until when I want to see the API or enter "the platform" of the project.
However, if I go in locally, this is by private IP, I can see the API screen.
The problem is a bit farfetched but it's there :(

**Comment by gabrielbasetti** [comment]
Good night!
I have reinstalled and configured everything again with the same luck :(
Recently, I decided to open the page and enter the mentioned section from the incognito tab and it WORKS.
I don't know what could be the problem that the page does not load correctly using my Google Chrome profile.
Perhaps this forum escapes the solution of this problem that I have, but if anyone can guide me where to look, I would appreciate it.
I leave this image with an error in the console, which does not appear in an incognito tab:

![image](https://user-images.githubusercontent.com/39503933/223288221-ee1b1ed0-cdef-46fd-ae7f-4756b40c04a2.png)
